doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -944,6 +949,22 @@ DDR                        OK        UserDefine
 ```
 
 show interface status - will show the NPU-DPU interface status also      <font>**`Executed on the switch`**</font>
+
+The internal DPU ports use role type in interfaces to indicate NPU-DPU Data Port Type as shown here. Check the type for "Ethernet224" in the "show interface status" output
+```
+{
+    "interfaces": {
+        "Ethernet224": {
+            "lanes": "2828,2829,2830,2831",
+            "index": "28,28,28,28",
+            "breakout_modes": {"1x200G": ["dpu-0"]},
+            "subport": "1",
+            "autoneg": "on",
+            "role": "Dpc"
+        },
+    }
+}
```

Comment:
@rameshraghupathy please specify which file will have this ? hwsku.json?

Reply:
Yes, hwsku.json.  This section has been updated to show the correct config and also mentioned the file name.